### ISSUE TITLE
QGIS Raster display is poorly calculated when changing projections

### ISSUE THREAD

**Issue by tbarsballe** [boundlessgeo/suite#850]
Raster bounds are badly generated for projections, especially when using "Zoom Full"
Example:
![qgis_raster_bounds_proj](https://cloud.githubusercontent.com/assets/9271497/5747133/8eda0d5e-9bea-11e4-93c4-dbe9d40d1f3a.png)

(Tested on OS X 10.10, QGIS 2.6.1)

**Comment by mweisman** [comment]
@dakcarto should we be moving/duping this ticket in the QGIS tracker? I'm not sure we have anyone who can fix this.

**Comment by dakcarto** [comment]
Yes, these types of QGIS-core-specific issues should be reported upstream instead.
